ci/jenkins/jobs/job-templates.yaml

```diff
@@ -267,6 +267,35 @@
         included-regions: '{included_regions}'
     publishers: '{publishers}'
     wrappers: '{wrappers}'
+- job-template:
```

Comment:
Is you PR fully ready? can you describe the details of this PR and what's we should expect after merging this PR?